velox/exec/prefixsort/benchmarks/PrefixSortAlgorithmBenchmark.cpp

```diff
@@ -25,6 +25,15 @@ using namespace facebook::velox;
 using namespace facebook::velox::exec;
 
 namespace {
+
+namespace {
+int initMemoryManager() {
```

Comment:
This doesn't look like the right fix.

Let's change 'bm' variable to `std::unique_ptr<ExchangeBenchmark>` and initialize it inside main().

Reply:
oh , I have tried naked pointer could not work,  but unique_ptr can~

done.